pom.xml

```diff
@@ -118,8 +124,8 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.1</version>
           <configuration>
-            <source>1.5</source>
-            <target>1.5</target>
+            <source>1.6</source>
+            <target>1.6</target>
```

Comment:
FWIW, this should result in a bigger version jump. In #24 I did just that (-> 1.1.0-SNAPSHOT) and this should suffice here too, IMO.

Reply:
I just updated Java version to align with the version required by the lib
if you want, I drop this part, because not really related

Reply:
Ah, yes - makes sense. Let's see what the actual maintainers think about this.